pkgs/tools/misc/ili2c/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, fetchFromGitHub, jdk, ant, makeWrapper, jre }:
+
+stdenv.mkDerivation rec {
+  name = "ili2c-${version}";
+  version = "4.7.11";
+
+  buildInputs = [ ant jdk makeWrapper ];
+
+  src = fetchFromGitHub {
+    owner = "claeis";
+    repo = "ili2c";
+    rev = name;
+    sha256 = "1a7130w3lp5rjdjfd7i1sibkp0mckihccrmc63y1np6cch8x40jj";
+  };
+
+  buildPhase = "ant jar";
+
+  installPhase =
+    ''
+      mkdir -p $out/share/${name}
+      cp $build/build/source/build/jar/ili2c.jar $out/share/${name}
+
+      mkdir -p $out/bin
+      makeWrapper ${jre}/bin/java $out/bin/ili2c \
+        --add-flags "-jar $out/share/${name}/ili2c.jar"
+    '';
+
+  meta = with stdenv.lib; {
+    description = "The INTERLIS Compiler";
+    longDescription = ''
+      Checks the syntactical correctness of an INTERLIS data model.
+    '';
+    homepage = https://www.interlis.ch/downloads/ili2c;
```

Comment:
Please quote.

Reply:
OK. When should URL values be quoted in nix and when not?

Reply:
This is a brand new thing. There is an RFC to deprecate unquoted urls. So if (more realistically when) that RFC is accepted the answer will be to always quote.

Reply:
Changed: https://github.com/NixOS/nixpkgs/compare/b6388f7be50dbb30e56b18acb727fcabd0379fb5..2aaa8876ff68e454ff60fc9a2b6d2174df6bb17a#diff-efa52252ac746605798baf0ce1e25f13R33